### ISSUE TITLE
Unique signature to avoid cheat detection

### ISSUE THREAD

**Issue by MatthewARoy** [Siterizer/new-world-fishing-bot#245]
https://smile.amazon.com/gp/help/customer/display.html?nodeId=GJSYGMDGB7NMYLKE&pop-up=1&sa-no-redirect=1

`Like most games, many Amazon Games games require installation of anti-cheat software on your machine to help ensure a fair environment for all players. The software monitors active processes on your device for hacks, bots, and other suspicious programs that could interfere with gameplay. It transmits this data and data about the game, your account, and your device to us and our service providers so we can detect and prevent cheating and other malicious activity. This information is subject to the Amazon Privacy Notice.
`

Reopening the issue, this will definitely be a concern and probably soon

**Comment by MatthewARoy** [comment]
New terms and conditions added on game launch

**Comment by DataSiens** [comment]
I'm upvoting this

**Comment by syntax001** [comment]
You sure this is new? Theyve been using EAC since the beginning, so if this clause wasnt there before, they mustve forgotten to add it, cause thats what EAC should have been doing since the beginning

**Comment by futureproof123** [comment]
what if you run the bot on a virtual desktop / remote desktop, connect to ur 'gaming pc' via teamviewer or anydesk or whatever, and let the bot operate clicking thru the teamviewer / anydesk, that way it appears as if you are just playing the game from a remote location.
It would also bypass the signature bs because the bot is running on a remote pc.

would this work? anyone have any insight into this

**Comment by syntax001** [comment]
> what if you run the bot on a virtual desktop / remote desktop, connect to ur 'gaming pc' via teamviewer or anydesk or whatever, and let the bot operate clicking thru the teamviewer / anydesk, that way it appears as if you are just playing the game from a remote location. It would also bypass the signature bs because the bot is running on a remote pc.
> 
> would this work? anyone have any insight into this

yes people are already running it on VMs. if the program isnt running on the same PC as the game, there is no way to "detect" it

**Comment by Siterizer** [comment]
Sorry, end of the support. Statement:
https://github.com/Siterizer/new-world-fishing-bot#new-world-fishing-bot-end-of-support
